packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts

```diff
@@ -197,15 +197,13 @@ export class ImportCard extends LitElement {
   @internalProperty() selectedDefaultOption: number = 0;
 
   async onUploadGLB() {
-    const files = await this.fileModal.open();
+    const files: any = await this.fileModal.open();
```

Comment:
What actual TS errors are you getting around with `any`? That's a pretty blunt tool, so it's better if we can be more delicate and retain more of it's type checking power.

Reply:
It just isn't recognizing the key "name", i,.e. file[0] is an object that has a key, so that should probably be cast to the correct type, but I'm not sure what that is